ClassLoader referenced unknown path: FileNotFoundException: /jacoco.exec: open failed: EROFS

Started showing up after adding jacoco task in #177 

```
com.blacklivesmatter.policebrutality W/System: ClassLoader referenced unknown path: /data/app/com.blacklivesmatter.policebrutality-2/lib/arm
com.blacklivesmatter.policebrutality W/System.err: java.io.FileNotFoundException: /jacoco.exec: open failed: EROFS (Read-only file system)
com.blacklivesmatter.policebrutality W/System.err:     at libcore.io.IoBridge.open(IoBridge.java:452)
com.blacklivesmatter.policebrutality W/System.err:     at java.io.FileOutputStream.<init>(FileOutputStream.java:87)
com.blacklivesmatter.policebrutality W/System.err:     at org.jacoco.agent.rt.internal_8ff85ea.output.FileOutput.openFile(FileOutput.java:67)
com.blacklivesmatter.policebrutality W/System.err:     at org.jacoco.agent.rt.internal_8ff85ea.output.FileOutput.startup(FileOutput.java:49)
com.blacklivesmatter.policebrutality W/System.err:     at org.jacoco.agent.rt.internal_8ff85ea.Agent.startup(Agent.java:122)
com.blacklivesmatter.policebrutality W/System.err:     at org.jacoco.agent.rt.internal_8ff85ea.Agent.getInstance(Agent.java:50)
com.blacklivesmatter.policebrutality W/System.err:     at org.jacoco.agent.rt.internal_8ff85ea.Offline.<clinit>(Offline.java:31)
com.blacklivesmatter.policebrutality W/System.err:     at com.blacklivesmatter.policebrutality.BrutalityIncidentApplication.$jacocoInit(BrutalityIncidentApplication.kt)
com.blacklivesmatter.policebrutality W/System.err:     at com.blacklivesmatter.policebrutality.BrutalityIncidentApplication.<init>(BrutalityIncidentApplication.kt)
com.blacklivesmatter.policebrutality W/System.err:     at java.lang.Class.newInstance(Native Method)
com.blacklivesmatter.policebrutality W/System.err:     at android.app.Instrumentation.newApplication(Instrumentation.java:996)
com.blacklivesmatter.policebrutality W/System.err:     at android.app.Instrumentation.newApplication(Instrumentation.java:981)
com.blacklivesmatter.policebrutality W/System.err:     at android.app.LoadedApk.makeApplication(LoadedApk.java:573)
com.blacklivesmatter.policebrutality W/System.err:     at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4680)
com.blacklivesmatter.policebrutality W/System.err:     at android.app.ActivityThread.-wrap1(ActivityThread.java)
com.blacklivesmatter.policebrutality W/System.err:     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1405)
com.blacklivesmatter.policebrutality W/System.err:     at android.os.Handler.dispatchMessage(Handler.java:102)
com.blacklivesmatter.policebrutality W/System.err:     at android.os.Looper.loop(Looper.java:148)
com.blacklivesmatter.policebrutality W/System.err:     at android.app.ActivityThread.main(ActivityThread.java:5417)
com.blacklivesmatter.policebrutality W/System.err:     at java.lang.reflect.Method.invoke(Native Method)
com.blacklivesmatter.policebrutality W/System.err:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
com.blacklivesmatter.policebrutality W/System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
com.blacklivesmatter.policebrutality W/System.err: Caused by: android.system.ErrnoException: open failed: EROFS (Read-only file system)
com.blacklivesmatter.policebrutality W/System.err:     at libcore.io.Posix.open(Native Method)
com.blacklivesmatter.policebrutality W/System.err:     at libcore.io.BlockGuardOs.open(BlockGuardOs.java:186)
com.blacklivesmatter.policebrutality W/System.err:     at libcore.io.IoBridge.open(IoBridge.java:438)
com.blacklivesmatter.policebrutality W/System.err: 	... 21 more
```

Apply the fix mentioned in https://github.com/jacoco/jacoco/issues/968

--- a/android-app/app/src/androidTest/resources/jacoco-agent.properties
+++ b/android-app/app/src/androidTest/resources/jacoco-agent.properties
@@ -1,0 +1,3 @@
+# Avoids FileNotFoundException: /jacoco.exec
+# https://github.com/jacoco/jacoco/issues/968
+output=none

--- a/android-app/app/src/main/resources/jacoco-agent.properties
+++ b/android-app/app/src/main/resources/jacoco-agent.properties
@@ -1,0 +1,3 @@
+# Avoids FileNotFoundException: /jacoco.exec
+# https://github.com/jacoco/jacoco/issues/968
+output=none

--- a/android-app/app/src/test/resources/jacoco-agent.properties
+++ b/android-app/app/src/test/resources/jacoco-agent.properties
@@ -1,0 +1,3 @@
+# Avoids FileNotFoundException: /jacoco.exec
+# https://github.com/jacoco/jacoco/issues/968
+output=none

